rdkit/__init__.py

```diff
@@ -59,8 +59,10 @@ def __next__(self):
 def __vect__iter__(vect):
     return VectIter(vect)
 
+VECT_WRAPS = {'MatchTypeVect', 'UnsignedLong_Vect', 'VectSizeT', 'VectorOfStringVectors'}
```

Comment:
@greglandrum these have the default SWIG names, so I had to hard code them.

Should be good to go now,